src/app/services/client/course.ts

```diff
@@ -57,6 +57,15 @@ export const getCourses = async () => {
   return coursesDocs;
 };
 
+export const getCoursesByIds = async (
+  courseIds: String[]
+): Promise<IdentifiableCourse[]> => {
+  const courses = courseIds.map((id) => getCourse(id));
```

Comment:
I think firestore should support filtering https://cloud.google.com/firestore/docs/query-data/queries#in_not-in_and_array-contains-any